sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/SimpleField.java

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.models;
+
+import com.azure.core.util.CoreUtils;
+
+import java.util.Objects;
+
+/**
+ * A helper Field model to build a simple {@link Field}.
+ */
+public class SimpleField extends FieldBase {
+    private boolean key;
+    private boolean facetable;
+    private boolean sortable;
+    private boolean filterable;
+    private boolean hidden;
+
+    /**
+     * Initializes a new instance of the {@link SimpleField} class.
+     *
+     * @param name The name of the field, which must be unique within the index or parent field.
+     * @param dataType The {@link DataType} of the {@link Field}.
+     * @throws NullPointerException when {@code name} is null.
+     */
+    public SimpleField(String name, DataType dataType) {
```

Comment:
Need an overload for whether this is also a collection (like you did in `ComplexField`).